docs/appstate.md

```diff
@@ -22,6 +22,9 @@ For more information, see [Apple's documentation](https://developer.apple.com/do
 
 To see the current state, you can check `AppState.currentState`, which will be kept up-to-date. However, `currentState` will be null at launch while `AppState` retrieves it over the bridge.
 
+You can remove line 7 & 24 in the functional component example if you don't want to see the AppState update from active to inactive (iOS).
```

Comment:
Ohh, since this comment is only related to the functional component please move it inside the `functional syntax` block.